server.go

```diff
@@ -1170,7 +1170,7 @@ func (s *Server) handleStream(t transport.ServerTransport, stream *transport.Str
 		trInfo.tr.SetError()
 	}
 	if unknownDesc := s.opts.unknownStreamDesc; unknownDesc != nil {
-		s.processStreamingRPC(t, stream, nil, unknownDesc, trInfo)
+		s.processStreamingRPC(t, stream, srv, unknownDesc, trInfo)
```

Comment:
I believe this is unnecessary and also kind of incorrect.

`srv.server` is supposed to be the service implementation for the method. But for unknown method, such implementation doesn't exist, and passing in a "random" pointer doesn't seem like a good idea.

The old behavior, passing in `nil` pointer, would make it clear that the implementation of the service doesn't exist.